lib/Sema/CSSolver.cpp

```diff
@@ -375,8 +377,10 @@ bool ConstraintSystem::simplify() {
       retireFailedConstraint(constraint);
       if (isDebugMode()) {
         auto &log = llvm::errs();
-        log.indent(solverState->getCurrentIndent() + 2) << ")\n";
```

Comment:
This would print an unbalanced brace if constrain was `ConstraintKind::Disjunction ||  ConstraintKind::Conjunction`

<img width="1138" alt="Screenshot 2022-11-11 at 17 43 54" src="https://user-images.githubusercontent.com/8717108/201457028-fba7d418-68d5-46f6-b882-f13b53c01bfe.png">